src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceOfDay = 40;
+  const price = days * priceOfDay;
+  const week = 7;
+  const fewDays = 3;
+  const bigSale = 50;
+  const smallSale = 20;
+
+  if (days >= week) {
+    return price - bigSale;
+  } else if (days >= fewDays) {
+    return price - smallSale;
+  } else if (days >= 1) {
+    return price;
```

Comment:
```suggestion
  } else {
    return price;
```
Для останньої умови можна вже просто else